app/theme/brave/BRANDING.dev

```diff
@@ -5,6 +5,6 @@ PRODUCT_SHORTNAME=Brave
 PRODUCT_INSTALLER_FULLNAME=Brave Installer
 PRODUCT_INSTALLER_SHORTNAME=Brave Installer
 COPYRIGHT=Copyright 2016 The Brave Authors. All rights reserved.
-MAC_BUNDLE_ID=org.brave.Brave.dev
+MAC_BUNDLE_ID=org.brave.Brave
```

Comment:
Hmm, to support multi channel install, we should have different bundle id.

Reply:
Ok meant to bring this up as well. I may need to dig back through why this was an issue but it came down to `MAC_BUNDLE_ID` set's the base for any other helper's that need to be signed as well.

Ran into issues when everything was prefixed by `org.brave.Brave.dev` (i.e org.brave.Brave.dev.BraveHelper). I remembered seeing somewhere that chrome doesn't actually modify these during build time, and always has them set to org.brave.Brave, but then during signing changes it to the correct value (will try to find references to this).

Also taking a look at the app id's in the chrome canary package it seems only the root app itself is using the channel postfix.

I'm unsure if this is the ideal way of doing things but, currently having it updated here:
https://github.com/brave/brave-core/pull/319/files#diff-4eaebbcd1dc0da11e3a3ce488290bfc6R350

Reply:
If this change doesn't affect multi channel install support, I'm fine with this PR.
I'll test it with this PR and share it.

Reply:
With first commit in this PR and below change, I finished dev channel app signing.
```
diff --git a/build/mac/BUILD.gn b/build/mac/BUILD.gn
index 0dddf74..d29b69a 100644
--- a/build/mac/BUILD.gn
+++ b/build/mac/BUILD.gn
@@ -35,7 +35,7 @@ action("sign_app") {
     rebase_path(_packaging_dir),
     keychain_db,
     mac_signing_identifier,
-    provisioning_profile,
+    rebase_path(provisioning_profile),
   ]
   deps = [
     "//brave:chrome_app",
```

Reply:
Also, all other BRANDING files should be modified like this file.

Reply:
updated